src/Microsoft.Health.Fhir.Core/Features/Search/SearchService.cs

```diff
@@ -27,6 +27,9 @@ public abstract class SearchService : ISearchService, IProvideCapability
         private readonly IBundleFactory _bundleFactory;
         private readonly IFhirDataStore _fhirDataStore;
 
+        // Value which is subtracted from Now when querying _history without _before specified
+        private readonly int historyCurrentTimeBuffer = -3;
```

Comment:
nit: give the variable name unit (e.g., InSeconds) or make it into TimeSpan for readability.